doc/source/manual/xtrabackup_sst.rst

```diff
@@ -49,208 +47,203 @@ Xbstream is highly recommended. Refer to :ref:`Xbstream v/s Tar <tar_ag_xbstream
-                                                                                                             
-.. option:: tca 
+Used to specify the data transfer format. The recommended value is ``transferfmt=socat`` because it allows for socket options, such as transfer buffer sizes. For more information, see `socat(1) <http://www.dest-unreach.org/socat/doc/socat.html>`_.
+
+.. option:: tca
 
-     :Description: CA file for openssl based encryption with socat.                                                   
-     :Type: Full path to CRT file (.crt).
+     :Example: tca=~/etc/ssl/certs/mycert.crt
+
+Used to specify the full path to the certificate authority (CA) file for ``socat`` encryption based on OpenSSL.
                           
 .. option:: tcert
+
+     :Example: tcert=~/etc/ssl/certs/mycert.pem
+
+Used to specify the full path to the certificate file in PEM format for ``socat`` encryption based on OpenSSL.
     
-    :Description: PEM for openssl based encryption with socat.                                                     
-    :Type:  Full path to PEM (.pem).
+.. note:: For more information about ``tca`` and ``tcert``, refer to http://www.dest-unreach.org/socat/doc/socat-openssltunnel.html. The ``tca`` is essentially the self-signed certificate in that example, and ``tcert`` is the PEM file generated after concatenation of the key and the certificate generated earlier. The names of options were chosen to be compatible with ``socat`` parameter names as well as with MySQL's SSL authentication. For testing you can also download certificates from `launchpad <https://bazaar.launchpad.net/~percona-core/percona-xtradb-cluster/5.5/files/head:/tests/certs/>`_.
+
+.. note:: Irrespective of what is shown in the example, you can use the same .crt and .pem files on all nodes and it will work, since there is no server-client paradigm here but a cluster with homogeneous nodes.
 
-.. note::
-    For tca and tcert, refer to http://www.dest-unreach.org/socat/doc/socat-openssltunnel.html for an example. The ``tca`` is essentially the self-signed certificate in that example, and ``tcert`` is the PEM file generated after concatenation of the key and the certificate generated earlier. The names of options were chosen so as to be compatible with socat's parameter' names as well as with MySQL's SSL authentication. For testing you can also download certificates from `launchpad <https://bazaar.launchpad.net/~percona-core/percona-xtradb-cluster/5.5/files/head:/tests/certs/>`_. **Note** that irrespective of what is shown in the example, you can use same crt and pem files on all nodes and it will work, since there is no server-client paradigm here but a cluster with homogeneous nodes.
-                                                                                                             
 .. option:: encrypt
 
-    :Values: 0,1,2,3
+    :Values: 0, 1, 2, 3
```

Comment:
Sorry, I couldn't find more info about encrypt=3 from Galera. I think you can proceed with existing change for now and in due-course once we get some more information we can further add to it.